### DISCUSSION TITLE
Guide V1 end of Q1

### DISCUSSION THREAD

**Discussion by GBKS** [BitcoinDesign/Guide#153] [Archived]
One of the goals we set at the start of the year was to have our first V1 of the guide ready for the public by the end of the first quarter.
- Is that something we still want to do?
- What do we expect from that first version?
- How can we get it done?

**Comment by pavlenex** [comment]
I think with recent sprints and better organization we're really getting things done faster, so I still think it's achievable. 

Personally, the completeness of  V1 to me would be:

- Introduction section finished
- Onboarding section finished
- Private Key Management finished
- Payments finished
- Principles/Foundation finished
- Clear process on how to get involved (Contribute section)
- Wallet UI Kit
- We have 1 project that we can work a bit closer with before v1 goes live

IMO the way to get to this stage, we have to stay consistent with sprints and focus on finishing things we've started instead of starting new ones.

**Reply by ConorOkus** [reply]
I agree with most of these, would add a caveat that "finished" is more of an acceptable for version 1 type of thing just to prevent anyone thinking it won't grow and change over time

**Reply by GBKS** [reply]
How do you see the Wallet UI Kit as part of the guide?

**Reply by pavlenex** [reply]
Wallet UI Kit connects lots of abstractions into something immediately usable. IMO that's the whole point of the guide. Give the foundational knowledge but do provide lots of material for people to immediately build UI's.

My two cents, is that it should be added to the guide. I'm a bit worried that if we start having all these separate "projects" they may loose on visibility, so at this stage I'd keep it in the guide.

**Comment by pavlenex** [comment]
A bit of update regarding the V1. We've discussed a bit on how the ideal V1 would look, and we agreed that we would try to wrap something we consider a decent V1 by the end of the [Sprint #2.](https://github.com/BitcoinDesign/Guide/milestone/3)

However, this won't mean the public launch just yet, as we feel there are a lot of moving parts we'd have to do first. We want to complete all the WIP chapters and then focus on polishing things up. Once the Sprint2 completes, it may be a good idea as we agreed, to reach out to a focused group of designers and people in the industry to get feedback.

**Comment by GBKS** [comment]
Picking up this discussion again based on our chat yesterday during sprint planning.

**Review parties**
Before getting to the launch, we need to ensure the guide content is of high quality and useful to our target audience. For that, we talked about shifting more focus towards inviting and working with reviewers. The bigger idea is that we could do this systematically. 

For example:
- We host one or two moderated guide review calls every week
- Initially we keep the circle smaller
- Later we invite technical experts to evaluate the content for accuracy (maybe also copy writing)
- Later expand to our target audiences (designers and product builders)

Conor and Johns basically started doing that the past 2 weeks spontaneously. Having a plan/schedule here would be very helpful for coordination, ensuring all content is reviewed, and that many others can easily participate.

For now, there are several tasks in [Sprint #3](https://github.com/BitcoinDesign/Guide/milestone/4) to review content. Makes sense to me that some of us start focusing harder on review. Question is just whether we turn this into a bigger, more organized, thing.

**IRL use**
The guide is meant to educate designers and inform product design & development. We don't want to launch the V1 to an empty room without a plan. So how do we transition smoothly from a small circle of people writing to a large grope of people using? Maybe we can create some theories on how the V1 will be super useful and already search for ways to start making them happen.

I see three general buckets:
1. Education, where a designer learns from the guide and then integrates those learnings in their work.
- Goal is to spread the content to more people
- The aforementioned review periods with our target audience could be very helpful
- We can reach out to designers and ask what problems they are currently dealing with
- We could also post more on social media to share the content wider and have more conversations
- We can host topical chats or workshops on Clubhouse, Twitch, Jitsi, etc

2. Resources (like case studies and the icon set) that embody guide content, which a designer or developer can directly use in their work
- Goal is to find projects who want to adopt the resources
- Zeus and Specter are good examples. In Zeus, the designers are using the icon set. In specter, there is designer/developer collaboration to integrate it. What other projects could we talk to?
- Could also promote the resources better (custom websites) and make them easier to integrate

3. Collaborations, where design community members work with a project and use or reference the guide
- Goal is to bring guide/design knowledge actively into projects by working with them (Zeus, Specter, LDK, what did I miss?)
- Could start more collaborations, but need to be sensitive that we can actually do the work to the level we want

In 2 and 3, the work is ideally heavily informed by the guide and references it, so it is a way of getting the guide out there - through practical application.

---

This ended up like a bit of a brain dump. In summary, it would be cool to celebrate a big fat 1.0 guide and proudly talk about it, and simultaneously be able to back up its usefulness by pointing at several efforts that have already benefited from it. In the end, the goal is that more people can use Bitcoin more easily. So IRL use matters. Yolo.

**Comment by pavlenex** [comment]
## Bitcoin Design V1 Update

Today we had a call about the state of the Bitcoin Design Guide. I'll try to summarize the discussion. Hoping other contributors can chime in as well. This is how I felt about the discussion:

1. We're currently trying to wrap up sections in [Sprint #4.](https://github.com/BitcoinDesign/Guide/milestone/5) in ~3 weeks.
2. Sprint 5 will be focused on the review and reaching out to people/community silently for early feedback.
3. By the end of Sprint 5, we should be ready to launch the V1.

## What do we consider a V1?

The consensus was that we need some closure and celebration. Hopefully, in 5 weeks, we'll have all the chapters complete in terms of content. By having a v1 out there, we'd be able to take a look at it from a different angle, gather feedback and take a step back and re-evaluate what's missing. We'll then evaluate if we want to expand current sections, or create new ones.

## How about V2?

The consensus was that in the V2, we could focus more on actually collaborating with projects, learning, and bringing back those learnings into the guide. Just putting the guide in the hands of the people and listening to the feedback.

**Reply by GBKS** [reply]
Good summary. There's of course a lot of nuance in this conversation, so anyone feel free to ask questions and share opinions.

Just a small addition on V2. The opposite approach of what Pavlenex described would be to look at the guide ourselves, find gaps and keep working at it. I know from the content I worked on, there's plenty I think should be added or improved. This would improve the guide, but not necessarily address what the needs are "out in the wild". So by starting with an external focus, our efforts are ideally more useful to the ecosystem.
